Can not import the library

**Bug description**
I can not import the package from node. I spent 10h.

**Expected behavior**
The package is imported and I can use the `parseFile` function.

**Audio file demonstrating the problem**
~/music-metadata   
❯ npm i music-metadata

added 17 packages, and audited 18 packages in 1s

8 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```                                              
~/music-metadata-test  
❯ node
Welcome to Node.js v18.4.0.
Type ".help" for more information.
> require('music-metadata')
Uncaught:
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/janwirth/music-metadata/node_modules/music-metadata/lib/index.js not supported.
Instead change the require of index.js in null to a dynamic import() which is available in all CommonJS modules.
    at __node_internal_captureLargerStackTrace (node:internal/errors:477:5)
    at new NodeError (node:internal/errors:388:5)
    at Module._extensions..js (node:internal/modules/cjs/loader:1139:19)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:834:12)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18) {
  code: 'ERR_REQUIRE_ESM'
}
```

@Borewit 
I use it from a very basic typescript project and the same is happening, I cannot use it and can't figure out what's going on.

Basic typescript project, running from the command line.

```

[INFO] 00:41:07 ts-node-dev ver. 2.0.0 (using ts-node ver. 10.9.1, typescript ver. 4.9.4)
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /music-match/node_modules/music-metadata/lib/index.js
require() of ES modules is not supported.
require() of /music-match/node_modules/music-metadata/lib/index.js from /music-match/src/music-match.ts is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /music-match/node_modules/music-metadata/package.json.

    at new NodeError (internal/errors.js:322:7)
    at Module._extensions..js (internal/modules/cjs/loader.js:1102:13)
    at require.extensions..jsx.require.extensions..js (/tmp/ts-node-dev-hook-9363314290809173.js:114:20)
    at Object.nodeDevHook [as .js] (/music-match/node_modules/ts-node-dev/lib/hook.js:63:13)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/music-match/src/music-match.ts:8:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
[ERROR] 00:41:09 Error: Must use import to load ES Module: /music-match/node_modules/music-metadata/lib/index.js
require() of ES modules is not supported.
require() of /music-match/node_modules/music-metadata/lib/index.js from /music-match/src/music-match.ts is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.

```

--- a/README.md
+++ b/README.md
@@ -25,6 +25,9 @@ The [`music-metadata`](https://github.com/Borewit/music-metadata) module is idea
 Module: version 8 migrated from [CommonJS](https://en.wikipedia.org/wiki/CommonJS) to [pure ECMAScript Module (ESM)](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
 The distributed JavaScript codebase is compliant with the [ECMAScript 2020 (11th Edition)](https://en.wikipedia.org/wiki/ECMAScript_version_history#11th_Edition_%E2%80%93_ECMAScript_2020) standard.
 
+> [!NOTE]
+> See also [CommonJS Backward Compatibility](#commonjs-backward-compatibility)
+
 This module requires a [Node.js ≥ 16](https://nodejs.org/en/about/previous-releases) engine.
 It can also be used in a browser environment when bundled with a module bundler.
 
@@ -764,6 +767,23 @@ Dependency list:
 - [@tokenizer-token](https://github.com/Borewit/tokenizer-token)
 - [peek-readable](https://github.com/Borewit/peek-readable)
 
+## CommonJS Backward compatibility
+
+For legacy CommonJS projects needing to load the `music-metadata` ESM module, you can use the `loadMusicMetadata` function:
+```js
+const { loadMusicMetadata } = require('music-metadata');
+
+(async () => {
+  // Dynamically loads the ESM module in a CommonJS project
+  const mm = await loadMusicMetadata();
+
+  const metadata = await mm.parseFile('/path/to/your/file');
+})();
+
+```
+
+> [!NOTE]
+> The `loadMusicMetadata` function is experimental and is not currently covered by any TypeScript typings.
 
 ## Frequently Asked Questions
 

--- a/lib/default.cjs
+++ b/lib/default.cjs
@@ -1,0 +1,5 @@
+// CommonJS core (default) entry point
+"use strict";
+module.exports = {
+  loadMusicMetadata: () => import('./core.js'),
+};

--- a/lib/node.cjs
+++ b/lib/node.cjs
@@ -1,0 +1,5 @@
+// CommonJS Node entry point
+"use strict";
+module.exports = {
+  loadMusicMetadata: () => import('./index.js'),
+};

--- a/package.json
+++ b/package.json
@@ -19,14 +19,23 @@
   "type": "module",
   "exports": {
     ".": {
-      "node": "./lib/index.js",
-      "default": "./lib/core.js"
+      "node": {
+        "import": "./lib/index.js",
+        "require": "./lib/node.cjs",
+        "types": "./lib/index.d.ts"
+      },
+      "default": {
+        "import": "./lib/core.js",
+        "require": "./lib/default.cjs",
+        "types": "./lib/core.d.ts"
+      }
     }
   },
   "types": "lib/index.d.ts",
   "files": [
     "lib/**/*.js",
-    "lib/**/*.d.ts"
+    "lib/**/*.d.ts",
+    "lib/*.cjs"
   ],
   "keywords": [
     "music",

